docs/sphinx/source/whatsnew/v0.11.1.rst

```diff
@@ -39,7 +35,6 @@ Enhancements
   obviating the call to scipy.optimize that was prone to runtime errors and minimizing
   computation. (:issue:`2171`, :issue:`2216`, :pull:`2217`).
```

Comment:
scipy.optimise was not prone to minimising computation... I think this is clearer
```suggestion
   thus minimizing computation and obviating the call to scipy.optimize, which was prone to runtime errors. (:issue:`2171`, :issue:`2216`, :pull:`2217`).
```